llama-index-networks/llama_index/networks/network/retriever.py

```diff
@@ -15,17 +16,36 @@ class NetworkRetriever(BaseRetriever):
     def __init__(
         self,
         contributors: List[ContributorRetrieverClient],
+        reranker: Optional[BaseNodePostprocessor] = None,
+        rerank: bool = False,
```

Comment:
Just curious, why we need both to provide a reranker and enable reranking?

Reply:
yeah, don't need both! will just use `reranker` if exists, then rerank :)